### PR TITLE
PLENTY-300 install php codesniffer

### PR DESCRIPTION
**Pull request by RimantasAdeoWeb**

### CODE REVIEW

File: src/Services/SearchService.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ class SearchService implements SearchServiceInterface
 {
     use \Plenty\Plugin\Log\Loggable;
 
-    CONST DEFAULT_ITEMS_PER_PAGE = 25;
```
Comment (TheKeymaster) [inline src/Services/SearchService.php:33]: Can't believe that PHP actually allows this :see_no_evil:

File: tests/Services/Search/ParametersHandlerTest.php [hunk 2]
```diff
@@ -466,12 +466,12 @@ protected function createCeresConfigObject(
         $defaultSortingSearch = '',
         $defaultSorting = ''
     ) {
-        return $config = (object) array(
-            'sorting' => (object) array(
+        return $config = (object) [
```
Comment (TheKeymaster) [inline tests/Services/Search/ParametersHandlerTest.php:469]: This is some hacky code right there. Can we use `stdClass` instead here?

File: .travis.yml [hunk 3]
```diff
@@ -13,4 +13,5 @@ install:
   - php composer.phar install
 
 script:
-  - composer test
+  - composer lint
+  - composer test
```
Comment (TheKeymaster) [inline .travis.yml:17]: Please add a newline at the end here.

**Review by TheKeymaster — CHANGES_REQUESTED** [review]

**Comment by RimantasAdeoWeb** [comment]
Sorry, had to force-push. Messed the branch up a bit on thursday trying to figure out why it wouldn't build (turned out not to be related to this after all)

**Review by TheKeymaster — APPROVED** [review]

**Pull request merged**